src/typeDefs/types.ts

```diff
@@ -81,15 +87,20 @@ export const types = gql`
     nameOfUser: String!
     nameOfOrg: String!
     amount: Float!
+    createdAt: DateTime
+    updatedAt: DateTime
   }
   type Advertisement {
-    _id: ID
+    _id: ID!
     name: String!
-    orgId: ID
+    orgId: ID!
```

Comment:
regarding `orgId` field take a look at this https://github.com/PalisadoesFoundation/talawa-api/pull/1585#discussion_r1445939073